Xamarin.Essentials/Barometer/Barometer.ios.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CoreMotion;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Barometer
+    {
+        static bool PlatformIsSupported => CMAltimeter.IsRelativeAltitudeAvailable;
+
+        static CMAltimeter altitudeManager;
+
+        static void PlatformStart()
+        {
+            altitudeManager = new CMAltimeter();
+            altitudeManager.StartRelativeAltitudeUpdates(NSOperationQueue.CurrentQueue, LocationManagerUpdatedHeading);
```

Comment:
This should be: Platform.GetCurrentQueue()

Reply:
Done